Fixed #34429 -- Allowed setting unusable passwords for users in the auth forms. (#16942)
django/django: docs/topics/auth/default.txt

.. class:: AdminPasswordChangeForm

>     A form used in the admin interface to change a user's password. This form
>     also allows the password to be set as an :meth:`unusable password
>     <django.contrib.auth.models.User.set_unusable_password>` effectively
>     barring the user from logging in.

Reviewer: ```suggestion
    A form used in the admin interface to change a user's password, including
    the ability to set an :meth:`unusable password
    <django.contrib.auth.models.User.set_unusable_password>`, effectively
    barring the user from logging in using a password-based authentication.
```

    .. versionchanged:: 5.1

>         Option to disable password-based authentication was added.

Reviewer: ```suggestion
        Option to disable (or re-enable) password-based authentication was added.
```

.. class:: AdminPasswordChangeForm

    A form used in the admin interface to change a user's password, including
    the ability to set an :meth:`unusable password
    <django.contrib.auth.models.User.set_unusable_password>`, effectively
    barring the user from logging in using password-based authentication.

    .. versionchanged:: 5.1

>         Option to disable (or re-enable) password-based authentication was

Reviewer 2: `reenable` is even in the `spelling_wordlist`:
```suggestion
        Option to disable (or reenable) password-based authentication was
```

You should see a link to "Users" in the "Auth"
section of the main admin index page. The "Add user" admin page is different
than standard admin pages in that it requires you to choose a username and

> password (or setting the password as unusable) before allowing you to edit the rest of

Reviewer: I forgot to mention that docs should always be wrapped at 79cols, so this line may need an adjustment.

You should see a link to "Users" in the "Auth"
section of the main admin index page. The "Add user" admin page is different
than standard admin pages in that it requires you to choose a username and
password (or setting the password as unusable) before allowing you to edit the
rest of the user's fields.

>     .. versionchanged:: 5.1
> 
>         Checkbox option to set an unusable password was added.

Reviewer: ```suggestion
    .. versionchanged:: 5.1

        Option to set an unusable password was added.
```

>         Option to disable password-based authentication was added.

Reviewer: ```suggestion
        Option to disable (or re-enable) password-based authentication was added.
```
